app/experimenter/experiments/views.py

```diff
@@ -104,6 +119,11 @@ class Meta:
         form = ExperimentFiltersetForm
         fields = ExperimentFiltersetForm.Meta.fields
 
+    def filter_search(self, queryset, name, value):
+        return queryset.annotate(
+            search=SearchVector("name", "short_description")
```

Comment:
I think the fields we wanna search should include:

- owner.email
- name
- slug
- short_description
- related_work
- addon_experiment_id
- pref_key
- public_name
- public_description
- objectives
- analysis
- analysis_owner
- engineering_owner
- bugzilla_id
- normandy_slug

I think the test you have is fine, you don't need to test against searching each of these fields, but check by hand to see if they work as expected.

Reply:
@jaredkerim, I had a bit of a discussion days ago with @mythmon about this, and he had brought up a few concerns about text searching so many fields that an experiment could be returned even though it really wasn't obvious why it was returned, with nothing in the experiment preview box obviously matching the search text. Does this worry you at all, or would you rather just have the wider search net thrown? One thing we might want to consider if we use the multiple fields is the concept of 'weighting' or 'ranking' the results (I've seen these mentioned in the `SearchVector` docs) so that the better matches float higher. Second thought: I was wondering if since we already have a filter dedicated to searching a field like owner email.. should still include that one? I definitely see why looking for owner email with the text search might be easier than using that owner email select filter.

Reply:
@uhlissuh Yeah I'm not worried about 'why' it matches being reflected, just that if someone throws an email in that field, we should return all the experiments that refer to that email, or any kind of slug/identifier, like normandy slug/addon id, etc.  

You don't want the opposite to happen, ie you search for an email/identifier, don't see the thing you're looking for in the search results, find it some other way and clearly right on the detail page is the term you searched for being clearly displayed, but wasn't included in the search fields.

If we bump into performance limitations we can think about that separately.

Also yes adding some kind of weighting would also be super cool!

Reply:
OK great, will implement this change to what we're filtering against and the other changes you requested! :)